packages/utils/src/date/index.ts

```diff
@@ -217,7 +217,7 @@ export default {
 	convertToTimeZone,
 	convertToUTC,
 	format,
-	options,
+	FORMAT,
```

Comment:
But you remove the options export?

Reply:
I don't want [options](https://github.com/Talend/ui/blob/master/packages/utils/src/date/index.ts#L197) to be exposed, only [FORMAT](FORMAT).
See my previous comment [here](https://github.com/Talend/ui/pull/3358#discussion_r662147649).

Reply:
understand 👍 seems fine if it's not used for now, if so, it will be a blocker